src/OpenTelemetry.Extensions.Enrichment.AspNetCore/OpenTelemetry.Extensions.Enrichment.AspNetCore.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimumSupportedVersion);$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
+    <Description>OpenTelemetry .NET SDK ASP.NET Core telemetry enrichment.</Description>
+    <NoWarn>$(NoWarn),CS1591</NoWarn>
+    <MinVerTagPrefix>Extensions.Enrichment.AspNetCore-</MinVerTagPrefix>
+    <Nullable>enable</Nullable>
+    <AnalysisLevel>latest-all</AnalysisLevel>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.0.0-rc9.14" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="$(RepoRoot)\src\OpenTelemetry.Internal\Guard.cs" Link="Includes\Guard.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\OpenTelemetry.Extensions.Enrichment\OpenTelemetry.Extensions.Enrichment.csproj" />
```

Comment:
I think that it should be switched to PackageReference when base package will be released.

Reply:
Yes, but now it has to be a `ProjectReference`, right?